conans/cli/commands/list.py

```diff
@@ -0,0 +1,165 @@
+import json
+
+from conans.cli.output import cli_out_write
+from conans.cli.command import conan_command, conan_subcommand, Extender, COMMAND_GROUPS
+from conans.client.output import Color
+from conans.util.dates import iso8601_to_str
+
+indentation = 2
+remote_color = Color.BRIGHT_BLUE
+recipe_color = Color.BRIGHT_WHITE
+reference_color = Color.WHITE
+
+
+def list_recipes_cli_formatter(results):
```

Comment:
I start thinking of we should keep all the `xxxx_formatter` functions in another specific module, e.g., `cli/formatters.py`

Reply:
Let's wait and see if we need it. Except the `json_formatter` function, the command and formatter functions are completely coupled, so it makes sense to me to keep them in the same module.

Reply:
I prefer the formatters to be close to their specific commands, there is little value in putting them together, as they can be very different, decoupled outputs, while for every command, the data structure is 90% of the implementation, so IMHO it makes sense to have formatters grouped by command.